Sources/MapboxNavigation/DayStyle.swift

```diff
@@ -259,6 +261,12 @@ open class DayStyle: Style {
             UITableView.appearance(whenContainedInInstancesOf: [StepsViewController.self]).sectionHeaderTopPadding = 0.0
         }
         #endif
+        
+        if #available(iOS 13.0, *) {
```

Comment:
Why not use one default color regardless of iOS version?

Reply:
Thanks for the suggestion, it was using the previous code for color. I'll change it default to the blue one.